notebooks/qg4Dvar_tutorial.ipynb

```diff
@@ -34,7 +34,7 @@
     "<br><br>\n",
     "<center>$y = H(x_{true}) + \\epsilon_{true}$ </center> \n",
     "<br>\n",
-    "in which $H(..)$ is the so-called observation operator that maps a state $x$ {\\em at the beginning of the time window} to observation space. Observation space in this case contains observations valid over the whole time window. This means that to compare the model state with the observations $H$ has first bring the model state to the observation time, and then extract from the state at that time the corresponding modelled observation. {\\em This means that $H$ now contains the model operator.} The $\\epsilon_{true}$ is the observation error. This error is due to the measurement process itself: every measurement will have an error related to the precision of the measurement. There is also another part to this error, called the representation error, that we will not discuss here as it is not essential for understanding 4DVar. <br><br>\n",
+    "in which $H(..)$ is the so-called observation operator that maps a state $x$ {\\em(?) at the beginning of the time window} to observation space. Observation space in this case contains observations valid over the whole time window. This means that to compare the model state with the observations $H$ has to first bring the model state to the observation time, and then extract from the state at that time the corresponding modelled observation. {\\em(?) This means that $H$ now contains the model operator.} The $\\epsilon_{true}$ is the observation error. This error is due to the measurement process itself: every measurement will have an error related to the precision of the measurement. There is also another part to this error, called the representation error, that we will not discuss here as it is not essential for understanding 4DVar. <br><br>\n",
```

Comment:
This `\em` markup doesn't seem to be getting rendered correctly. Not sure what is intended or correct. I just marked with a `(?)` here and elsewhere.

Reply:
Note I also added the word `to` to this line